ui/src/config/section/account.js

```diff
@@ -225,11 +225,10 @@ export default {
       message: 'message.delete.account',
       dataView: true,
       disabled: (record, store) => {
-        return record.id !== 'undefined' && store.userInfo.accountid === record.id
+        return (record.id !== 'undefined' && store.userInfo.accountid === record.id)
```

Comment:
I'm aware this isn't directly related to the PR, but I think the original condition doesn't make much sense. It should be `record?.id !== undefined`. In fact, it can be simplified to:

```suggestion
        return store.userInfo.accountid === record?.id
```

The original check would only be valid if `record.id` could somehow hold the string `"undefined"`, which seems very unlikely. However, this should first be tested to confirm the behavior before applying the changes.

Reply:
Yes, it's tested and committed. Thanks for noticing!

Reply:
Indeed,  simplifying it to store.userInfo.accountid === record?.id makes it cleaner and more accurate.